setup_jenkins.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+sed 's/DOCKER_REGISTRY=.*/DOCKER_REGISTRY=ghcr.io\/candig/; s/WORKING_DIR=.*//' etc/env/example.env > .env
+
+grep -q "finished init-conda" $PWD/progress.txt
+if [ $? -ne 0 ]; then
+  echo "need to re-run setup"
+  rm $PWD/progress.txt
+  make bin-all
+  make init-conda
+  . $PWD/bin/miniconda3/etc/profile.d/conda.sh
+  conda activate candig
+  pip install -U -r $PWD/etc/venv/requirements.txt
+else
+  echo "already set up"
+fi
```

Comment:
Don't understand why this is needed?

Reply:
You don't like reassurance that the other half of a conditional was reached?